Resources/libraries/supported-libraries-foundation-hudson-thames.html

```diff
@@ -0,0 +1,4 @@
+<div class="section-example-container">
+<pre class="python">
+
```

Comment:
We need to manually add the content here. 
Activate H&T and get the output of `pip list`. E.g. for `Pomegranate`
```
/Foundation-Pomegranate/bin/activate
pip list
```